CHANGES.md

```diff
@@ -2,3 +2,8 @@
 =========================
 
  * First release!
+
+2016-08-10, Version 2.0.0-alpha.1
```

Comment:
The contents in `CHANGES.md` is generated by our release tooling, please revert this change.